packages/jest-runtime/src/index.ts

```diff
@@ -2226,6 +2248,7 @@ export default class Runtime {
       getTimerCount: () => _getFakeTimers().getTimerCount(),
       isMockFunction: this._moduleMocker.isMockFunction,
       isolateModules,
+      isolateModulesAsync,
```

Comment:
Isn’t it better to simply inline:

```suggestion
      isolateModulesAsync: this.isolateModulesAsync,
```

Reply:
Done in 36c138e2a5cd8796f67a78ae624e5d581ee93d0e